media-source/MediaSource-addSourceBuffer_null.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset='utf-8'>
+  <title>MediaSource#addSourceBuffer(null) and MediaSource#addSourceBuffer("")</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<div id="log"></div>
+
+<script>
+async_test(function(t) {
+    var mediaSource = new MediaSource();
+    mediaSource.addEventListener('sourceopen', function(e) {
+        var sourceBuffer = null;
+        t.step(function() {
+            assert_throws(new TypeError(),
+                          function() {
+                              sourceBuffer = mediaSource.addSourceBuffer(null);
+                          },
+                          'Set null');
```

Comment:
null is tested already in mediasource-addsourcebuffer.html, though that test needs to be corrected to expect TypeError. I'll make that correction separately.